src/components/TermBox.vue

```diff
@@ -36,9 +41,12 @@ export default class TermBox extends Vue {
 
 <style lang="scss">
 .wikibase-termbox { // container - need a strong selector chain to reliably override reset css
-	display: flex;
 
 	.wikibase-termbox { // for use as a prefix
+		&__primary {
+			display: flex;
```

Comment:
Now we are flexing all around - not a bad thing but maybe we can specify do this once on a high level (instead of time again with using extra classes) and just add controlling features in places? Queue reading session.

Reply:
I'm pretty sure we need this on a per-flexy-element basis. This one is here so that the edit pen sits next to it.

Reply:
Agreed. "enables a flex context for all its _direct_ children." https://css-tricks.com/snippets/css/a-guide-to-flexbox/